Source/Documentation/Manual/operation.rst

```diff
@@ -1855,7 +1859,22 @@ is inserted into the activity file following the line::
 
 (note that the number in the brackets may be different), then AI trains will 
 blow their horn at level crossings for a random time between 2 and 5 
-seconds.The level crossing must be defined as such in the MSTS route editor. 
+seconds. The level crossing must be defined as such in the MSTS route editor. 
+
+If line::
+
+    ORTSAIHornAtCrossings ( 1 )
+
+is followed by line::
```

Comment:
```suggestion
If the following line is inserted::
```
I don't believe the order of the properties matters. Same goes for `ORTSAIHornAtCrossings ( 1 )` and `NextActivityObjectUID ( 32768 )` mentioned just above this section. If so, could we clarify that here?

Reply:
The position of the 1 or two added lines with respect to NextActivityObjectUID ( 32768 ) must be the one defined, else the MSTS AE crashes when opening such an activity. It was not easy to find a position for the added line (it was only one up to the last change you suggested) that didn't cause crashes to MSTS. It is true that the reciprocal order of the two added lines doesn't matter, but I prefer my wording to yours, because you don't specify where the other line must be inserted. 
Moreover there  is no advantage for the activity writer to know that he can add the new line before ORTSAIHornAtCrossings ( 1 ). So I don't agree to change what I have written.

Reply:
Oh! I had completely forgotten about the activity editor. This is really interesting information that I think deserves to be in the manual (in the form of another PR).

> ...and I also think that one should calibrate the requests for refinements to the importance of them.

I do apologize if this point seems pedantic, but it's still untrue information, and that didn't sit well with me. All I wanted was some clarification, which you gave in spades, so thanks very much.